File: C# Dasar/Contoh Input & Output/input & output.cs
using System;

class main
{
    static void Main(String[] args)
    {
        int i;

        Console.Write("Masukkan Data berupa angka : ");
        i = int.Parse(Console.ReadLine());

        Console.WriteLine($"Output dari Data yang diinputkan : {i}");
    }
}

File: Kotlin/Contoh Input & Output/Input & Output.kt
import java.util.Scanner

fun main(args: Array<String>) {

    val terminalInput = Scanner(System.`in`)
    print("Masukkan Data berupa angka : ")

    var i:Int = terminalInput.nextInt()

    println("Output dari Data yang diinputkan :  $i")
}